package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "clr-addons",
-  "version": "10.1.9",
+  "version": "10.1.10",
```

Comment:
Raising clarity versions is at least a minor release, because that version also needs to be updated in client code.
Please raise to 10.2.0